crates/iota-json-rpc-types/src/iota_transaction.rs

```diff
@@ -2113,18 +2113,18 @@ impl From<InputObjectKind> for IotaInputObjectKind {
 
 #[derive(Debug, Serialize, Deserialize, JsonSchema, Clone)]
 #[serde(rename = "TypeTag", rename_all = "camelCase")]
-pub struct IotaTypeTag(String);
+pub struct IotaTypeTag(pub String);
```

Comment:
I think I'd rather have a getter/AsRef/etc than a public field, to avoid having it inadvertently changed